cargo-apk/src/main.rs

```diff
@@ -1,82 +1,75 @@
 use cargo_apk::{ApkBuilder, Error};
-use cargo_subcommand::Subcommand;
-use std::process::Command;
+use cargo_subcommand::{Args, Subcommand};
+use clap::Parser;
+
+#[derive(Parser)]
+struct Cmd {
+    #[clap(subcommand)]
+    apk: ApkCmd,
+}
+
+#[derive(clap::Subcommand)]
+enum ApkCmd {
+    /// Helps cargo build apk's for android
+    Apk {
+        #[clap(subcommand)]
+        cmd: ApkSubCmd,
+    },
+}
+
+#[derive(clap::Subcommand)]
+enum ApkSubCmd {
+    /// Checks that the current package builds without creating an apk
+    Check {
+        #[clap(flatten)]
+        args: Args,
+    },
+    /// Compiles the current package and creates an apk
```

Comment:
I think all doc comments should be written in first person, ie. `Compile` and `build` without `-s`.